examples/tensorflow/segmentation/evaluation.py

```diff
@@ -90,8 +93,12 @@ def get_dataset_builders(config, num_devices):
     val_builder = COCODatasetBuilder(config=config,
                                      is_train=False,
                                      num_devices=num_devices)
-
-    return val_builder
+    config_ = config.deepcopy()
+    config_.batch_size = val_builder.batch_size
+    calibration_builder = COCODatasetBuilder(config=config_,
+                                             is_train=True,
+                                             num_devices=1)
+    return val_builder, calibration_builder
```

Comment:
Added calibration dataset for segmentation sample to initialize compression on eval.